### PR TITLE
COVID-19-metrics-webapp

### PR DESCRIPTION
**Pull request by Debas-31**
- [x] Use React components.
- [x] Use React props.
- [x] Use React Router.
- [x] Connect React and Redux.
- [x] Write integration tests with a React testing library.
- [x] Use styles in a React app.
- [x] Apply React best practices and language style guides in code.
- [x] Use store, actions, and reducers in React.
- [x] The project is a single-page application (SPA) built with React and Redux.
- [x] The data retrieved from the API should be stored in the Redux store.
- [x] You should filter the data that you retrieve from the API using a Filter stateless component.
- [x] Every page (the main page and the pages for each item) should have a unique route within the SPA.
- [x] The project should be deployed and accessible online ( a link to the online version of my app here is the [README file](https://github.com/Debas-31/metrics-webapp/blob/dev/README.md) ).

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { useSelector } from 'react-redux';
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import Header from './Header';
+
+const HomePage = () => {
```
Comment (TedLivist) [inline src/components/HomePage.js:7]: - [ ] Your Homepage is missing the descriptive banner section as contained in the original design. Kindly add it. 
- [ ] Also ensure the homepage contains information describing the data your app is showcasing.
- [ ] Kindly ensure these changes are reflected on the live demo

  Given design | Your implementation
  :---:|:---:
  debas-origin-home | debas-own-home

File: src/components/DetailsPage.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            </h2>
+            <h2 className="Name">
+              <span>Active : </span>
+              {state.active.toLocaleString()}
+            </h2>
+            <h2 className="Name">
+              <span>Tests : </span>
+              {state.tests.toLocaleString()}
+            </h2>
+            <h2>{}</h2>
```
Comment (TedLivist) [inline src/components/DetailsPage.js]: - [ ] I would recommend you follow the given design. Kindly add the alternating colors and the appropriate spacing between elements as we have in the template.
- [ ] Also ensure this change reflects on the live demo.

  Behance Design | Your Design
  :---:|:---:
  debas-orig-detail | debas-own-detail

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Debas-31,

Good job so far!
✔️ Good video presentation
✔️ Integration tests are written with React Testing Library
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
#### _Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/DetailsPage.js (hunk 2 above)
Comment (Debas-31) [reply src/components/DetailsPage.js]: Thank you for the feedback, @TedLivist. I have implemented the changes recommended.

Comment (DeliceLydia) [reply src/components/DetailsPage.js]: well addressed this

File: src/components/HomePage.js (hunk 1 above)
Comment (DeliceLydia) [reply src/components/HomePage.js:7]: Good job!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @Debas-31, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### You Rock!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**